packages/testcases/View Guided Answers Content.md

```diff
@@ -0,0 +1,79 @@
+
+Pre Req: Guided Answers Extension is installed
+
+
+### Open Guided Answers (SBAS and VSCODE)
+
+**GIVEN** the user opens in the 'Command Pallette' in SBAS/VSCODE
+**WHEN** the user enters 'SAP: Open Guided Answers'
+**THEN** 'SAP:Open Guided Answers' appears in the command palette drop down
+
+
+**GIVEN** 'SAP: Open Guided Answes' appears in the command paletted dropdown 
+**WHEN** The user selects 'SAP : Open Guided Answers' 
+**THEN** the Guided Answers Extension opens in the view port showing: 
+
+    a. A blank home page 
+    b. Empty search input field
+
+**GIVEN** the Guided Answer Extension is opened
+**WHEN** the makes the view port smaller 
+**THEN** the Guided Answers Extension adapts:
+  
+    a.The search input field falls under the SAP Guided Answers Extension 
+
+### Search Guided Answers Content
+
+**GIVEN** the Guided Answer Extension is opened
+**WHEN** the user enters search crtieria
```

Comment:
- Spell Check: `critieria` instead of `crtieria`
- For this test case, instead of ad-hoc search criteria, can you recommend a specific search text like `SAP Fiori tools` so that we know the search works correctly